napari/layers/shapes/shape_list.py

```diff
@@ -681,15 +672,15 @@ def to_masks(
                 s.to_mask(mask_shape, zoom_factor=zoom_factor, offset=offset)
                 for s in self.shapes
             ]
-        elif shape_type not in self._types.keys():
+        elif shape_type not in shape_classes.keys():
             raise ValueError(
                 """shape_type not recognized, must be one of
                          "{'line', 'rectangle', 'ellipse', 'path',
```

Comment:
you mean the error message shouldn't have the types in them? removed

Reply:
no, I mean you should include it as something like ```f'must be one of {set(shape_classes)}```

on another note I just realized that this is a literal string. this should also be changed since literal strings should almost never be used in-line as they will capture indentation which is undesirable.

Reply:
fixed